mw/diag/dtc/database/dtc_database.hpp

```diff
@@ -25,15 +25,19 @@ namespace dtc {
 
 class DtcDatabase {
  private:
-  std::unordered_map<uint16_t, DtcDatabaseElement> errors_{};
+ /**
+  * @brief kluczem jest id błędu
+  * 
+  */
+  std::unordered_map<uint16_t,DtcDatabaseElement> errors_{};
   core::logger::Logger logger_;
+  uint16_t error_id;
   uint16_t active_errors;
 
  public:
   DtcDatabase();
-  core::ErrorCode AddError(
-      uint16_t service_id, uint16_t error_id, std::string details,
-      DtcErrorStatus_t status = DtcErrorStatus_t::kCreated);
+  core::ErrorCode AddError(uint16_t dtc_error_code_,std::string details_,
+                            uint8_t status_=0x0);
   core::ErrorCode ManResetError(uint16_t error_id);
   core::ErrorCode AutoResetError(uint16_t error_id);
```

Comment:
to error_id to to samo co dtc_error_code_ ?

Reply:
nie